tests/core/test_read_csv.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+
+import polars as pl
+from polars.testing import assert_frame_equal
+from pytest import fixture, raises
+
+from seedcase_sprout.core.read_csv import read_csv
+
+
+@fixture
+def data_path(tmp_path) -> Path:
+    return tmp_path / "data.csv"
+
+
+def test_throws_error_if_file_not_found(data_path):
+    """Should throw an error if the file cannot be found."""
+
+    with raises(FileNotFoundError):
+        read_csv(data_path)
+
+
+def test_throws_value_error_if_file_not_csv(tmp_path):
+    """Should throw a ValueError if the file is not a CSV file."""
+    path = tmp_path / "not-data.txt"
```

Comment:
```suggestion
    path = tmp_path / "data.txt"
```
Why is this called no-data? :)